dashboard/app/models/stage.rb

```diff
@@ -75,6 +75,32 @@ def summarize
         levels: script.script_levels.to_a.select{|sl| sl.stage_id == id}.map(&:summarize),
     }
 
+    # Go through all levels.  If we find an assessment with a per_page value,
+    # then it's a long assessment: we assume it's the final level in the
```

Comment:
we should probably enforce this instead of assuming it (throwing an exception during seed should be sufficient)